### PR TITLE
[#3740] Update dotnet samples to target Net 6 - Bot essentials samples

### PR DESCRIPTION
**Pull request by erquirogasw**
Addresses # 3740
#minor

## Description
This PR updates the Bot essentials samples to target .NET6

## Specific Changes
- Updated the following samples to target .NET6
  - Echo-bot
  - Welcome-user
  - Multi-turn-prompt
  - Using-cards
  - Using-adaptive-cards
  - Suggested-actions
  - Core-bot
  - NLP-with-orchestrator
  - Handling-attachments
  - Timex-resolution
  - Complex-dialog
  - State-management
- README files updated to target .NET6

## Testing

Echo-bot
Screenshot 2022-05-11 105738

Welcome-user 
Screenshot 2022-05-11 103156

StateManagement and AdaptiveCards working
1a8ff05d-8cad-4388-81aa-aeef7c6733a7

HandlingAttachments and SuggestedAction working
f283d767-18e1-4134-9e5b-40a063668bb2

Multi-turn-prompt working
Screenshot 2022-05-11 110625

Using-cards working
Screenshot 2022-05-11 120218

Core-bot working
image (80)

NLP-with-orchestrator working
Screenshot 2022-05-12 161808

Timex-resolution working
image (81)

Complex-dialog working
Screenshot 2022-05-11 152115

### CODE REVIEW

File: samples/csharp_dotnetcore/14.nlp-with-orchestrator/README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ This bot uses Orchestrator to route user utterances to multiple LUIS models and
 This sample **requires** prerequisites in order to run.
 - Install latest supported version of [Visual C++ Redistributable](https://support.microsoft.com/en-gb/help/2977003/the-latest-supported-visual-c-downloads)
 - Install latest [Bot Framework Emulator](https://github.com/microsoft/BotFramework-Emulator/releases)
-- [.NET Core SDK](https://aka.ms/dotnet-core-applaunch?framework=Microsoft.AspNetCore.App&framework_version=3.1.0&arch=x64&rid=win10-x64) version 3.1
+- [.NET SDK](https://aka.ms/dotnet-core-applaunch?framework=Microsoft.AspNetCore.App&framework_version=3.1.0&arch=x64&rid=win10-x64) version 6.0
```
Comment (ceciliaavila) [inline samples/csharp_dotnetcore/14.nlp-with-orchestrator/README.md]: Check this link, you can change it for the ones in the other readmes.

**Review by sw-joelmut — APPROVED** [review]

**Review by ceciliaavila — CHANGES_REQUESTED** [review]
Just a small change in one of the readme files. Also, in the testing section of the PR, it'd be better to have screenshots of the bots working instead of the build.

File: samples/csharp_dotnetcore/14.nlp-with-orchestrator/README.md (hunk 1 above)
Comment (sw-joelmut) [reply samples/csharp_dotnetcore/14.nlp-with-orchestrator/README.md]: Also validate the link its redirecting to the right version
image

Comment (erquirogasw) [reply samples/csharp_dotnetcore/14.nlp-with-orchestrator/README.md]: Fixed. Thank you!
image

**Review by ceciliaavila — APPROVED** [review]

**Review by sw-joelmut — CHANGES_REQUESTED** [review]
Core bot tests project is missing here

**Comment by erquirogasw** [comment]
> Core bot tests project is missing here

Fixed. Thank you!

**Review by sw-joelmut — APPROVED** [review]

**Pull request closed without merging**